src/store/store.js

```diff
@@ -9,8 +9,11 @@ const configureStore = () => {
     rootReducer,
     compose(
       applyMiddleware(sagaMiddleware),
-      window.__REDUX_DEVTOOLS_EXTENSION__ &&
-        window.__REDUX_DEVTOOLS_EXTENSION__()
+      compose(
+        applyMiddleware(sagaMiddleware),
+        window.__REDUX_DEVTOOLS_EXTENSION__ &&
+          window.__REDUX_DEVTOOLS_EXTENSION__()
+      )
```

Comment:
Why did you add a second compose and applyMiddleware(sagaMiddleware),? There is compose at line 10

Reply:
fixed